package.json

```diff
@@ -60,6 +60,9 @@
     "pregenerate-vulnerability-report": "npm run create-dependency-sbom-lists && npm run snyk-test && npm run scan-node-js",
     "generate-vulnerability-report": "mongodb-sbom-tools generate-vulnerability-report --snyk-reports=.sbom/snyk-test-result.json,.sbom/node-js-vuln.json --dependencies=.sbom/dependencies.json,.sbom/node-js-dep.json --fail-on=high > .sbom/vulnerability-report.md",
     "create-vulnerability-tickets": "mongodb-sbom-tools generate-vulnerability-report --snyk-reports=.sbom/snyk-test-result.json,.sbom/node-js-vuln.json --dependencies=.sbom/dependencies.json,.sbom/node-js-dep.json --create-jira-issues",
+    "precreate-static-analysis-report": "npm run create-dependency-sbom-lists",
+    "create-static-analysis-report": "mongodb-sbom-tools fetch-codeql-results --sarif-dest=.sbom/codeql.sarif.json --dependencies=.sbom/dependencies.json --exclude-repos=mongodb-js/kerberos",
```

Comment:
A book keeping mention, if you find yourself working on the `mongodb-sbom-tools` project we could update that README with some of these new commands https://github.com/mongodb-js/devtools-shared/tree/main/packages/sbom-tools

Reply:
https://github.com/mongodb-js/devtools-shared/pull/341